maintainers/maintainer-list.nix

```diff
@@ -4431,6 +4431,12 @@
     githubId = 5525217;
     name = "Hunter Jones";
   };
+  hleboulanger = {
+    email = "hleboulanger@protonmail.com";
+    name = "Harold Leboulanger";
+    github = "thbkrhsw";
+    githubId = 33122;
+  };
```

Comment:
Please put this into a separate commit with the message `maintainers: add NAME` and sort it alphabetically